lib/node_modules/@stdlib/constants/float32/half-pi/README.md

```diff
@@ -0,0 +1,147 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# HALF_PI
+
+> One half times the mathematical constant [π][pi].
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var HALF_PI = require( '@stdlib/constants/float32/half-pi' );
+```
+
+#### HALF_PI
+
+One half times the mathematical constant [π][pi].
+
+```javascript
+var bool = ( HALF_PI === 1.5707963 );
+// returns true
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- TODO: better example -->
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var HALF_PI = require( '@stdlib/constants/float32/half-pi' );
+
+console.log( HALF_PI );
+// => 1.5707963
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/constants/float32/half_pi.h"
+```
+
+#### STDLIB_CONSTANT_FLOAT32_HALF_PI
+
+Macro for one half times the mathematical constant [π][pi].
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+</section>
+
+<!-- /.examples -->
+
+</section>
+
+<!-- /.c -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
```

Comment:
For new packages, always remove the contents of "related", leave the `<section>` tags. As mentioned in the comment, this section is managed by external tooling.